src/FreqWindow.cpp

```diff
@@ -1239,7 +1239,8 @@ void OnPlotSpectrum(const CommandContext &context)
 
    if( VetoDialogHook::Call( freqWindow ) )
       return;
-   freqWindow->Show(true);
+   if (!freqWindow->Show(true))
+       return;
    freqWindow->Raise();
```

Comment:
But does this only appear to fix it?  Is the un-raised window just hiding behind the project window which you can move aside?

Reply:
Good thinking, but I tested it and it seems like it really fixes it. Could @chinakov confirm ?

Reply:
Yes, the issue has been fixed.